program_info/org.polymc.PolyMC.metainfo.xml.in

```diff
@@ -22,44 +22,36 @@
       <li>See logs and other details easily</li>
       <li>Kill Minecraft in case of a crash/freeze</li>
       <li>Isolate minecraft instances to keep everything clean</li>
+      <li>Install mods directly from the launcher</li>
     </ul>
-    <p>For flatpak users:</p>
-    <p>In flatpak, all java versions that are required by Minecraft are included.</p>
-    <p>
-      If using a Hybrid-Graphics device, you can use the
-      prime-run
-      script as a wrapper command to run Minecraft using the dedicated graphics card.
-    </p>
   </description>
   <screenshots>
     <screenshot type="default">
       <caption>The main PolyMC window</caption>
-      <image type="source" width="802" height="639">https://i.imgur.com/q2GcDo4.png</image>
+      <image type="source" width="1011" height="994">https://polymc.github.io/polymc.org/assets/img/screenshots/LauncherDark.png</image>
```

Comment:
These URLs could change in the future (we might rename the website repo). Maybe just put the screenshots into `program_info/screenshots` and create permalinks to it?

Reply:
sure, I can do that
however it would be nice not having them in two places. Maybe I should reference commit names? Or create a branch in that repo with the current naming

Reply:
Or we can make sure that we rename the repository now. Then you can generate permalinks there. Let me ping the maintainers.

Reply:
github will redirect it anyways I think not sure if it redirects files aswell. But yeah having it renamed now would be nice